dashboard/app/assets/javascripts/header.js.erb

```diff
@@ -91,6 +93,26 @@ dashboard.buildHeader = function (stageData, progressData, currentLevelId) {
   $('.header_popup').click(function (e) {
     e.stopPropagation(); // Clicks inside the popup shouldn't close it
   });
+  $('.header_popup_close').click(hideHeaderPopup);
+
+  $(window).resize(function () {
+    if ($('.header_popup').is(':visible')) {
```

Comment:
A pattern I've used before (after finding discussed online) is to only do the actual resize 1 second or so after the final resize message fires.  This avoids a lot of DOM modifications while the user is dragging the window around.  Might not be necessary here, but if it were an expensive operation you can imagine some low power machines might actually struggle to show the resize continuously.